records_mover/session.py

```diff
@@ -41,6 +42,33 @@ def _infer_scratch_s3_url(session_type: str) -> Optional[str]:
     if "SCRATCH_S3_URL" in os.environ:
         return os.environ["SCRATCH_S3_URL"]
 
+    # TODO: do I want this in 'app.ini'?
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker

Reply:
Seems a little generic, but this seems to be config_resolver's default, and I guess having a directory named records_mover leaves room for additional files inside, so not sure I have a better idea!